parser/parser.y

```diff
@@ -9103,6 +9104,20 @@ SelectStmtLimit:
 		$$ = &ast.Limit{Count: $3.(ast.ExprNode)}
 	}
 
+SelectStmtLimitSimple:
+	"LIMIT" Int64Num
+	{
+		$$ = &ast.LimitSimple{Offset: 0, Count: $2.(int64)}
+	}
+|	"LIMIT" Int64Num ',' Int64Num
+	{
+		$$ = &ast.LimitSimple{Offset: $2.(int64), Count: $4.(int64)}
+	}
+|	"LIMIT" Int64Num "OFFSET" Int64Num
+	{
+		$$ = &ast.LimitSimple{Offset: $4.(int64), Count: $2.(int64)}
+	}
+
```

Comment:
Why not `AdminStmtLimitOpt`? It is not a select stmt.

Reply:
Modified.